components/needs_selector.js

```diff
@@ -93,7 +93,7 @@ NeedsSelector.propTypes = {
   selectedNeeds: PropTypes.object,
   t: PropTypes.func,
   theme: PropTypes.object,
-  clearNeeds: PropTypes.object
+  clearNeeds: PropTypes.func
```

Comment:
I introduced this bug. Thanks for catching that!

Reply:
I should have merged master first again and I think we should have been good